<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientChatWidget</class>
 <widget class="QWidget" name="ClientChatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTextEdit, QListWidget, QListView{
border-width: 2px;
     border-style: outset;
     border-color: #b87c23;
}
QLabel, QPushButton, QLineEdit{
 font-family: times, Times New Roman, times-roman, georgia, serif;
	color: white;
	margin: 0;
	padding: 0px 0px 0px 0px;
	letter-spacing: 0px;
	font-weight: bold;
}
QMainWindow{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #525a61, stop:1 #6a747d);
}
QPushButton{
border-width: 2px;
     border-style: outset;
     border-color: #61f2b7;
	 background-color: #2fc286;
}


QGroupBox, QLineEdit, QMenuBar{
	border-width: 2px;
    border-style: outset;
    border-color: #b87c23;
	font-family: times, Times New Roman, times-roman, georgia, serif;
	color: black;
	margin: 0;
	padding: 0px 0px 0px 0px;
	letter-spacing: 0px;
	font-weight: bold;

}
QGroupBox::title{
 	font-family: times, Times New Roman, times-roman, georgia, serif;
	color: white;
	margin: 0;
	padding: 0px 0px 0px 0px;
	letter-spacing: 0px;
	font-weight: bold;
	subcontrol-origin: margin;
	bottom: 5px;
}

QSplitter::handle{
background:transparent;
}
QSplitter::handle:hover{
cursor: default;
}
</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="lstMessages"/>
      </item>
      <item>
       <widget class="QWidget" name="wdgSend" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Message:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="InMessage"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnSend">
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextEdit" name="textUser"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Insernt user name</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
